src/ontology/phenio.Makefile

```diff
@@ -3,4 +3,34 @@
 ## If you need to customize your Makefile, make
 ## changes here rather than in the main Makefile
 
+# Replace the assembly of phenio-full.owl with a
+# similar process including subq reconstruction
 
+# Constants
+SUBQ_QUERY_PATH=                $(SPARQLDIR)/subq_construct.sparql
+SUBQ_QUERY_RESULT_PATH=         $(TMPDIR)/$(ONT)-full_subqs_queryresult.tmp.owl
+UPDATE_QUERY_PATH=              $(TMPDIR)/subq_update.sparql
+
+# Full file assembly
+$(TMPDIR)/$(ONT)-full.owl: $(SRC) $(OTHER_SRC) $(IMPORT_FILES)
+	$(ROBOT) merge --input $< \
```

Comment:
add `merge $(patsubst %, -i %, $(OTHER_SRC))` after this.

ODK does not presume you actually want to release all components in the full release -  but it should probably be an option. Feel free to make an ODK issue for this ("Make option for merging all components/imports into full release").

Reply:
Feel free to borrow from monarch.owl, but do not merge anything into master yet: https://github.com/monarch-initiative/monarch-ontology/blob/master/Makefile

Applying these things will _force_ it to work.

Chris would probably now say:

There is no need to `reason`.

But this is not true since we are using base, not base plus.

Reply:
I absolutely abhorr this stuff: https://github.com/monarch-initiative/monarch-ontology/blob/master/Makefile#L39